cluster/aws/util.sh

```diff
@@ -352,6 +352,34 @@ function wait-for-instance-running {
   done
 }
 
+function allocate-elastic-ip {
+  $AWS_CMD allocate-address --domain vpc --output text | cut -f3
+}
+
+function assign-ip-to-instance {
+  local ip_address=$1
+  local instance_id=$2
+  local elastic_ip_allocation_id=$($AWS_CMD describe-addresses --public-ips $ip_address --output text | cut -f2)
+  $AWS_CMD associate-address --instance-id $master_instance_id --allocation-id $elastic_ip_allocation_id > /dev/null
+  local association_result=$?
```

Comment:
This is going to go badly, because we set errexit at the top of [./cluster/kube-up.sh](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/kube-up.sh#L23). This means that if the result is != 0 the script will exit right away before you get to check the return value. Elsewhere we set a var and then do `|| var="failed"` or something and then check the value of var after the command finishes.